### PR TITLE
Carboxes quality improved. All images are saved as PNG format

### PR DESCRIPTION
**Pull request by EstebanMz**
All of carboxes has a resolution of at least 256x256, with the exception of the following cars:

ADVANCED: Weaver; SEMI-PRO: BHV 1; CLOCKWORK: Clockwork Baja, Clockwork Stream, Clockwork Trial.

This is due to those cars only have a 128x128px carbox, what was done was an upscale and apply a bit of sharpness to the existing files.

Aside from the carboxes, some track images files the filesize of some track pictures were reduced.

### CODE REVIEW

**Comment by BGMP** [comment]
Hey, thanks for the PR!

I had been thinking on adding a [rake task](https://github.com/Re-Volt-America/rva_cars/blob/master/Rakefile) with this procedure to have all images be converted automatically at the [rva_cars](https://github.com/Re-Volt-America/rva_cars) repository. The carboxes/ repo is generated from the current car pack, which we will change eventually, so it'd be really cool if I could automate this.

What did you do in order to convert all images to the normalised format?

File: assets/img/carboxes/desktop.ini [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ViewState]
+Mode=
+Vid=
+FolderType=Generic
```
Comment (BGMP) [inline assets/img/carboxes/desktop.ini]: Uhh, this probably shouldn't be here, right?

**Review by BGMP — CHANGES_REQUESTED** [review]
All other changes look good!

**Comment by EstebanMz** [comment]
> Hey, thanks for the PR!
> 
> I had been thinking on adding a [rake task](https://github.com/Re-Volt-America/rva_cars/blob/master/Rakefile) with this procedure to have all images be converted automatically at the [rva_cars](https://github.com/Re-Volt-America/rva_cars) repository. The carboxes/ repo is generated from the current car pack, which we will change eventually, so it'd be really cool if I could automate this.
> 
> What did you do in order to convert all images to the normalised format?

Opens Photoshop
Drag a BMP file
Save as... (Ctrl+Shift+S) and change format to PNG
Before confirm changes remove ".png" from file name and overwrite the file
Repeat

**Comment by EstebanMz** [comment]
Can you confirm everything is OK now before I close the pull?

**Review by BGMP — APPROVED** [review]
Looks good to go

**Pull request merged**